source/release-notes.txt

```diff
@@ -10,6 +10,22 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.39.1
+----------------
+
+*Released August 9, 2023*
+
+New Features:
+
+* Show insight for unbound array (:issue:`COMPASS-6836`)
```

Comment:
I wonder if we could ask the Compass team for more detail about this first item. I am not sure from the small description or the ticket what exactly this means.

Reply:
Fixed